tests/e2e/commandse2e/command.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package commandse2e
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+)
+
+type TestFlags map[string]string
+
+// CommandGroup represents the different command groups available in the CLI
+type CommandGroup string
+
+const (
+	BlockchainCmd CommandGroup = "blockchain"
+)
+
+var avalancheBinaryPath = "./bin/avalanche"
+
+var GlobalFlags = map[string]interface{}{
+	"local":             true,
+	"skip-icm-deploy":   true,
+	"skip-update-check": true,
+}
+
+// TestCommand tests a CLI command with flag inputs
+func TestCommand(commandGroup CommandGroup, command string, args []string, testFlags TestFlags) (string, error) {
```

Comment:
why not just receiving the command string instead of commandGRoup + command? we have some
commands without group